src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -404,18 +405,26 @@ class TestErrorAggregator extends FlatSpec with Matchers with BeforeAndAfterAll
   }
 
   "display version" can "be undefined" in {
-    import spark.implicits._
-    val messages = TestUtils.generateMainMessages(
-      1, None, None, "displayVersion" :: Nil
-    ).map(_.toByteArray).seq
-
-    val df = ErrorAggregator.aggregate(spark.sqlContext.createDataset(messages).toDF, raiseOnError = false,
-      ErrorAggregator.defaultDimensionsSchema, ErrorAggregator.defaultMetricsSchema,
-      ErrorAggregator.defaultCountHistogramErrorsSchema)
-
-    // 1 for each experiment (there are 2), and one for a null experiment
-    df.where("display_version IS NULL").collect().length should be (3)
-    // should be no cases where displayVersion is null for this test case
-    df.where("display_version <> NULL").collect().length should be (0)
+    // silence intentional http failed logs
+    val httpSinkLogger = Logger.getLogger("HttpSinkLogger")
+    val prevLogLevel = httpSinkLogger.getLevel
+    httpSinkLogger.setLevel(Level.ERROR)
+    try {
```

Comment:
I changed this log level in the code instead of `log4j.properties` because it seems like these kind of warnings would be useful in other contexts where they aren't intentional: 
<pre>
18/05/29 13:46:52 WARN HttpSinkLogger: Failed request: http://localhost:9876/httpapi?k=v, last status code: 666
</pre>